### ISSUE TITLE
folder_exclude_patterns doesn't work on sub-directories

### ISSUE THREAD

**Issue by Siyfion** [robcowie/SublimeTODO#44]
ie. Assuming I have this in my `.sublime-project` file:

```
"settings": {
  "todo": {
    "folder_exclude_patterns": [
      "node_modules",
      "tests/lib"
    ]
  }
}
```

Only the `node_modules` folder is excluded, the `tests/lib` folder still reports all it's todos, etc.

**Comment by dpashkevich** [comment]
Hm, this may be related to the issue I'm experiencing
https://github.com/robcowie/SublimeTODO/pull/12#issuecomment-13544680

If you add the same exclude patterns to your global `User preferences`, does the plugin start excluding `tests/lib`?

**Comment by Siyfion** [comment]
@dpashkevich nope, doesn't _seem_ to work either way, no matter where they are defined.

**Comment by domdoescode** [comment]
Also got this issue, using Ubuntu 12.04

**Comment by emero** [comment]
+1

**Comment by akempes** [comment]
The script doesn't allow folder paths but only the folder names. Add just "lib" to the "folder_exclude_patterns".

Sadly this will exclude **all** lib folder throughout the whole project...

**Comment by dpashkevich** [comment]
Got it, thanks for clarifying

**Comment by redbar0n** [comment]
+1

**Comment by redbar0n** [comment]
@akempes @dpashkevich actually, it does say it allows folder paths:

"The path may be relative to the project directory, or a fully qualified path."

https://www.sublimetext.com/docs/2/projects.html

**Comment by redbar0n** [comment]
This might help some get the folder_exclude_patterns to work:

http://stackoverflow.com/questions/14203944/sublime-text-folder-exclude-patterns-stopped-working
1. Close the project.
2. delete the .sublime-workspace file.
3. Reopen.

**Comment by gpascale** [comment]
Looks like this is a pretty issue, but I just ran into it too. I'll work around by renaming my folders but it's pretty annoying to have to do that. I can't seem exclude the folder "js" without also excluding "src/js". I tried putting "./js" as the exclude pattern but that doesn't do anything.

**Comment by jonathanrdelgado** [comment]
@gpascale, this plugin has been deprecated for  https://github.com/jonathandelgado/SublimeTodoReview, just fyi.

**Comment by robcowie** [comment]
Indeed it has. I still get quite a bit of activity on this repo. I'm not sure what the best course of action is; Delete this repo?

**Comment by jonathanrdelgado** [comment]
@robcowie I believe you can do a redirect, https://github.com/blog/1508-repository-redirects-are-here

I may be completely wrong about the use though. Nice to see you around, by the way. Hope all is well with you.

**Comment by gpascale** [comment]
Oh whoops, I thought I was commenting on the main SublimeText repo. I now
see that the issue I'm referring
to wasn't even the main issue here but just a comment in the thread.
Disregard :)

**Comment by robcowie** [comment]
@jonathandelgado I think the opportunity to use a redirect has been missed. I should have transferred ownership of this repo. As it is, there is no implied relationship between this and the new repo and it doesn't seem possible to create arbitrary redirects. Ah well.

All is well thanks; Spending most of my coding time these days with Apache Spark. A blessing and a curse in equal measure!
